### DISCUSSION TITLE
Editing the 3D segmentation results

### DISCUSSION THREAD

**Discussion by Harris91-lee** [Project-MONAI/MONAI#7823] [General]
Hello everyone, 
i have already trained my model and the result is okay somehow, however due to false positive or false negative i thought of what if i can edit manually the segmented result. 
Does Monai or pytorch support something like that?

**Comment by KumoLiu** [comment]
Hi @Harris91-lee, you can edit it by using itk. Also, you can refer to MONAILabel: https://github.com/Project-MONAI/MONAILabel.

Thanks.
